### PR TITLE
switch to maven

### PR DESCRIPTION
**Pull request by AWare**
switch to maven as bintray is being turned off

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,33 +6,12 @@ This repository contains the Thrift schema required for interaction with [guardi
...
-3. You will need an API key.
-  1. Go to your profile
-  2. Click the Edit button near your profile name (top left).
-  3. At the bottom of the list on the left will be the API key section containing your key. Keep this key handy for the next step.
-4. Setup your username/API key locally.
-  1. In this project run `sbt bintrayChangeCredentials`
-  2. Enter your username and API key as prompted.
-  3. This will save your creds locally and you shouldn't need to change them unless you refresh your API key.
-
+Please follow standard department advice on publishing to maven.
```
Comment (jonathonherbert) [inline README.md]: Possible to add a link here?

**Comment by jonathonherbert** [comment]
Is this published and tested? Happy to approve if so ✨

File: README.md (hunk 1 above)
Comment (twrichards) [reply README.md]: done

**Comment by twrichards** [comment]
It's on sonatype servers...
image
... but still waiting for it to appear in Maven Central

**Review by jonathonherbert — APPROVED** [review]
I can see PRs that reference this PR building successfully, so presume this is now out and OK – lmk if not the case!

**Pull request merged**